homeassistant/components/here_travel_time/sensor.py

```diff
@@ -121,21 +121,28 @@ def __init__(
         )
         self._attr_has_entity_name = True
 
+    async def _async_restore_state(self) -> None:
+        """Restore state."""
+        if restored_data := await self.async_get_last_sensor_data():
+            self._attr_native_value = restored_data.native_value
+
     async def async_added_to_hass(self) -> None:
         """Wait for start so origin and destination entities can be resolved."""
+        await self._async_restore_state()
         await super().async_added_to_hass()
 
         async def _update_at_start(_):
             await self.async_update()
 
         self.async_on_remove(async_at_start(self.hass, _update_at_start))
 
-    @property
-    def native_value(self) -> str | float | None:
-        """Return the state of the sensor."""
+    def _handle_coordinator_update(self) -> None:
```

Comment:
Please decorate the method with `@callback` as in the parent coordinator entity class.